.github/workflows/build_and_test.yml

```diff
@@ -7781,6 +7812,7 @@ jobs:
     - name: check build configuration
       shell: cmd
       run: |
+        set SSL_DIR=%CD%/openssl3
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/environment-variables

Apparently we can do this inside the job and it will have it for the entire job:
```

env:
      SSL_DIR:%CD%/openssl3
```